eslint.config.js

```diff
@@ -0,0 +1,9 @@
+import neostandard, { resolveIgnoresFromGitignore } from 'neostandard'
+
+export default neostandard({
+  ts: true,
+  env: ['mocha'],
+  ignores: [
+    ...resolveIgnoresFromGitignore(),
```

Comment:
Nice util `resolveIgnoresFromGitignore`, I had to make one from `eslint` helpers